projects/openmote-b/00std_spi/00std_spi.c

```diff
@@ -117,7 +122,7 @@ void    spi_txrx(uint8_t*     bufTx,
     // SPI is now busy
     spi_vars.busy             =  1;
 
-    // lower CS signal to have slave listening
+    // lower CS signal to have slave listening. FSS is managed through software, not hardware.
```

Comment:
The SSI peripheral is able to control the FSS signal. We could change the code to do that.